### PR TITLE
optimize createBlock - generate the labels map only once.

### PR DESCRIPTION
**Pull request by krasi-georgiev**
This reduces the CPU time in half.

cc @codesome 

Signed-off-by: Krasi Georgiev <kgeorgie@redhat.com>

### CODE REVIEW

**Comment by brian-brazil** [comment]
This code isn't doing the same thing, each created series would now have the same labels.

**Comment by codesome** [comment]
Agree with @brian-brazil

**Comment by krasi-georgiev** [comment]
yes that is right but with maps the order is randomised so it is essentially different series.

Anyway I added some more code to add one unique label so now it is clear that it creates unique series.

File: block_test.go [hunk 1]
```diff
@@ -102,15 +102,19 @@ func genSeries(totalSeries, labelCount int, mint, maxt int64) []Series {
 	}
 	series := make([]Series, totalSeries)
 
+	lbls := make(map[string]string, labelCount)
+	var lastKey string
+	for len(lbls) < labelCount {
+		lastKey := randString()
```
Comment (codesome) [inline block_test.go]:
```suggestion
		lastKey = randString()
```

**Comment by krasi-georgiev** [comment]
it is not in an immediate  needed so will close and revisit if needed.

**Pull request closed without merging**